Pear/Onboarding/AnswerPromptViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  AnswerPromptViewController.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/28/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class AnswerPromptViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let titleLabel = UILabel()
+    private let questionLabel = UILabel()
+    private let responseTextView = UITextView()
+    private let characterCountLabel = UILabel()
+    private let saveButton = UIButton()
+
+    // MARK: - Private Data Vars
+    private var prompt: Prompt?
+    private var index: Int?
+    private weak var delegate: AddProfilePromptDelegate?
+
+    init(prompt: Prompt, delegate: AddProfilePromptDelegate, index: Int) {
+        super.init(nibName: nil, bundle: nil)
```

Comment:
You can move `super.init` to the bottom of this function - so then `prompt`, `index`, and `delegate` will no longer have to be optionals